pkg/k8sutil/client.go

```diff
@@ -20,6 +20,8 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
+// NewClientsetFromFile creates a new Kubernetes Client set object from the given
```

Comment:
nit:  the first and this commit should be merged into a single commit.

Reply:
Nah, I'm not sure about that. Any specific arguments why to merge them instead of keeping them separate? It's 2 separate tasks, one is to rename the function, so refactoring (and ignoring . This one is fixing the linter. Perhaps one reason to merge them would be to make linter succeed per commit, though it's not a big deal as it's only the linter.

Reply:
I merged it anyway, so keep linter happy even between the commits